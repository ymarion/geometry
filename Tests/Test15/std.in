# Loading the file into the model
LOAD loadFile.txt
UNDO
# Checking nothing has been loaded
LIST
#
EXIT
