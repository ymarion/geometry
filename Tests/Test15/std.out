# Load:
OK
# Creation of the 5 figures from loadFile.txt
OK
OK
OK
OK
OK
# Undo:
OK
# Deletion of the 5 figures
OK
OK
OK
OK
OK
# Value of LIST:
EMPTY
